integration/simulation/network/inmemory.go

```diff
@@ -94,7 +94,7 @@ func (n *basicNetworkOfInMemoryNodes) Create(params *params.SimParams, stats *st
 	for _, m := range n.ethNodes {
 		t := m
 		go t.Start()
-		time.Sleep(params.AvgBlockDuration / 20)
+		time.Sleep(params.AvgBlockDuration)
```

Comment:
Before this change, every node would log this error: `err="could not fetch block at requested height=1 - not found"`